FeedReader/FeedReader.csproj

```diff
@@ -1,31 +1,31 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;net452</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net452;netcoreapp2.1</TargetFrameworks>
     <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>key.snk</AssemblyOriginatorKeyFile>
 	<Title>CodeHollow.FeedReader</Title>
     <Authors>Armin Reiter</Authors>
     <Company>Reiter ITS</Company>
     <Product>CodeHollow.FeedReader</Product>
-    <Description>FeedReader is a .net standard 1.3 library used for reading and parsing RSS and ATOM feeds. Supports RSS 0.91, 0.92, 1.0, 2.0 and ATOM. Developed because existing libraries do not work with different languages, encodings or have other issues. Library tested with multiple languages and feeds.</Description>
+    <Description>FeedReader is a .net standard 2.0 library used for reading and parsing RSS and ATOM feeds. Supports RSS 0.91, 0.92, 1.0, 2.0 and ATOM. Developed because existing libraries do not work with different languages, encodings or have other issues. Library tested with multiple languages and feeds.</Description>
 	<Copyright>Copyright 2018</Copyright>
     <PackageLicenseUrl>https://github.com/codehollow/FeedReader/blob/master/LICENSE</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/codehollow/FeedReader/</PackageProjectUrl>
     <RepositoryUrl>https://github.com/codehollow/FeedReader/</RepositoryUrl>
-    <Version>1.1.4</Version>
+    <Version>1.1.5</Version>
```

Comment:
Was not sure if you bump versions for this project...I can remove this if need be